quarkchain/cluster/master.py

```diff
@@ -1064,9 +1064,10 @@ def __log_summary(self):
     async def __init_cluster(self):
         await self.__connect_to_slaves()
         self.__log_summary()
-        if not self.__has_all_shards():
-            Logger.error("Missing some shards. Check cluster config file!")
-            return
+        if self.slave_pool:
```

Comment:
line 1065 to line 1071 are all specific to qkcrpc. your condition check can be added on top of all of them